nilearn/connectome/connectivity_matrices.py

```diff
@@ -284,13 +330,17 @@ class ConnectivityMeasure(BaseEstimator, TransformerMixin):
             "covariance", "precision"}, optional
         The matrix kind.
 
+    vectorize : bool, optional
```

Comment:
I mean that I didn't read the diff properly :$. Sorry.

The description of the argument could be a tiny bit longer, that said, explaining what you mean by vectorizing.